plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -127,6 +128,9 @@ export const AboutContent = ({ entity }: Props) => {
           <Chip key={t} size="small" label={t} />
         ))}
       </AboutField>
+      <AboutField label="Badges" gridSizes={{ xs: 12, sm: 6, lg: 4 }}>
+        <EntityBadgesField />
+      </AboutField>
```

Comment:
as mentioned elsewhere, maybe this should be found through [the header](https://github.com/backstage/backstage/blob/385cb44b3fbb8aa12b84de679e52c916958cd3e3/plugins/catalog/src/components/EntityPageLayout/EntityPageLayout.tsx#L105)'s dropdown instead

Reply:
Indeed, I'll remove this.